cmd/init-osm-controller/init-osm-controller_test.go

```diff
@@ -45,10 +45,10 @@ func TestCreateDefaultMeshConfig(t *testing.T) {
 	}
   },
   "certificate": {
-	"serviceCertValidityDuration": "24h"
+	"serviceCertValidityDuration": "23h"
```

Comment:
Why are we changing this value to 23h?

Reply:
Because 24h is the default value in the CRD. Changing it to 23h in test can ensure the option is set by preset-mesh-config.